tensorflow/core/kernels/dml_cwise_ops.cc

```diff
@@ -707,6 +707,10 @@ REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(SigmoidGrad, y* x*(1 - x),
                                            kNchwDimensionCount)
 REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(TanhGrad, y*(1 - x * x),
                                            kNchwDimensionCount)
+REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(SqrtGrad, y * 0.5f / x,
+                                           kNchwDimensionCount)
+REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(RsqrtGrad, y * (-0.5f * x) * (x * x),
```

Comment:
Do we know if identity(x) * x * x faster than identity(x^3)?

Reply:
Actually, doesn't DML pow use repeated squaring?

Reply:
In lieu of evidence for either case, I'm inclined to leave it as-is. :)